thetis/options.py

```diff
@@ -245,6 +245,12 @@ def __init__(self):
 
         Bottom stress is :math:`\tau_b/\rho_0 = -g \mu^2 |\mathbf{u}|\mathbf{u}/H^{1/3}`
         """
+        self.wd_alpha = None
+        r"""Coefficient or None: Wetting-drying parameter :math:`\alpha`
+
+        Used in bathymetry displacement function that ensures positive water depths. Unit is meters.
+        Default is None, which disables wetting and drying.
```

Comment:
I'd prefer to do it now.  Don't perpetuate existing bad practice, it just makes it harder to fix later.